trinity/sync/beam/importer.py

```
@@ -493,6 +494,8 @@ def partial_speculative_execute(
     Get an argument-free function that will trigger missing state downloads,
     by executing all the transactions, in the context of the given header.
     """
+
+    @suppress_exceptions(BrokenPipeError)
```

Can we do this in a way where we still get `ERROR` logs but just as a single line?

I don't believe silencing these exception is the right thing but I expect it to take a few iterations before we can really tackle the problem correctly .